admin/display/menu.php

```diff
@@ -44,6 +48,27 @@ function add_pro_submenu() {
 			$url,
 		);
 	}
+
+	/*
+	 * Opens Upgrade link in menu as new tab.
+	 *
+	 * @since 3.0
+	 */
+	function upgrade_link_aioseop_menu_new_tab() {
```

Comment:
With these functions, it would probably better to reduce it to 1 function named `admin_enqueue_scripts`. It would reduce the amount of code, and the purpose would be more clear when looking over a class's method names. I was trying to suggest it with my idea, and forgot to mention to apply it here as well.

Reply:
Done.